pandas/core/strings.py

```diff
@@ -1116,9 +1116,8 @@ def str_split(arr, pat=None, n=None):
 
     Returns
     -------
-    split : Series/Index or DataFrame/MultiIndex of objects
-        Type matches caller unless ``expand=True`` (return type is DataFrame or
-        MultiIndex)
+    split : Series, Index, DataFrame or MultiIndex of objects
```

Comment:
Since you are only returning one item you don't need to list the variable name, so get rid of "split : ". Also chop " of objects" off the end

Reply:
done :+1:

Reply:
@WillAyd I was trying to resolve for coverage and I think I messed up with PR. Does everything look good to you? :sweat_smile: